pkg/cmd/convert.go

```diff
@@ -75,7 +75,7 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 
 		o, err := decoder.Decode(data)
 		if err != nil {
-			return nil, fmt.Errorf("could not unmarsha data for file '%s': %s", file, err)
+			return nil, fmt.Errorf("could not unmarshal data for file '%s': %s", file, err)
```

Comment:
Could you please make it a different commit as it has no relation to *Added support for providing replicas*?